### PR TITLE
Folijet tests for parallel

### PR DESCRIPTION
**Pull request by TetianaParanich**
https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/1294/allure/

### CODE REVIEW

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/e2e/requests/edit-request.cy.js [hunk 1]
```diff
@@ -45,7 +45,6 @@ describe('ui-requests: Request: Edit requests. Make sure that edits are being sa
       cancelledDate: new Date().toISOString(),
```
Comment (penfriend) [inline cypress/e2e/requests/edit-request.cy.js:45]: 'oldRulesText' is assigned a value but never used. Allowed unused vars must match /React/u

File: cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js [hunk 2]
```diff
@@ -221,10 +221,6 @@ describe('ui-users-loans: renewal failure because loan has reached maximum renew
       InventoryInstance.deleteInstanceViaApi(secondInstanceIds.instanceId);
     });
 
-    cy.updateCirculationRules({
```
Comment (penfriend) [inline cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js:224]: 'rulesDefaultString' is assigned a value but never used. Allowed unused vars must match /React/u

File: cypress/support/fragments/checkout/modals/limitCheckOut.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ import { Button, Modal, including, HTML } from '../../../../../interactors';
 
 export default {
   verifyErrorMessage:(quantityItem) => {
+    cy.wait(1500);
```
Comment (penfriend) [inline cypress/support/fragments/checkout/modals/limitCheckOut.js:5]: why do you need this waiter?

File: cypress/e2e/requests/edit-request.cy.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/e2e/requests/edit-request.cy.js:43]: Deleted

File: cypress/e2e/requests/request-types-filter.cy.js [hunk 4]
```diff
@@ -50,7 +50,6 @@ describe('ui-requests: Make sure that request type filters are working properly'
     userIds.forEach(id => {
       Users.deleteViaApi(id);
     });
-    Requests.updateCirculationRulesApi(oldRulesText);
```
[the remark this answers is not in the document]
Comment (TetianaParanich) [reply cypress/e2e/requests/request-types-filter.cy.js:53]: Deleted

File: cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js:224]: Deleted

**Pull request merged**